packages/web/src/pages/audio-rewards-page/components/WalletManagementTile.tsx

```diff
@@ -0,0 +1,242 @@
+import { formatWei } from 'common/utils/wallet'
+import MobileConnectWalletsDrawer from 'components/mobile-connect-wallets-drawer/MobileConnectWalletsDrawer'
+import { isMobile } from 'utils/clientUtil'
+
+import TokenHoverTooltip from './TokenHoverTooltip'
+import styles from './WalletManagementTile.module.css'
+
+const messages = {
+  receiveLabel: 'Receive $AUDIO',
+  sendLabel: 'Send $AUDIO',
+  audio: '$AUDIO',
+  manageWallets: 'Manage Wallets',
+  connectWallets: 'Connect Other Wallets',
+  totalAudio: 'Total $AUDIO',
+  buyAudio: 'Buy $AUDIO',
+  checkoutWithCoinbase: 'Checkout with Coinbase',
+  showAdvanced: 'Show Advanced',
+  hideAdvanced: 'Hide Advanced',
+  advancedOptions: 'Advanced Options',
+  buyWith: 'Buy with',
+  buyWithCoinbase: 'Buy with Coinbase'
+}
+const IS_NATIVE_MOBILE = process.env.REACT_APP_NATIVE_MOBILE
+
+const AdvancedWalletActions = () => {
+  const balance = useSelector(getAccountBalance) ?? (new BN(0) as BNWei)
+  const hasBalance = balance && !balance.isZero()
+  const dispatch = useDispatch()
+  const [, openTransferDrawer] = useModalState('TransferAudioMobileWarning')
+
+  const mobile = isMobile()
+  const onClickReceive = useCallback(() => {
+    if (mobile) {
+      openTransferDrawer(true)
+    } else {
+      dispatch(pressReceive())
+    }
+  }, [dispatch, mobile, openTransferDrawer])
+
+  const onClickSend = useCallback(() => {
+    if (mobile) {
+      openTransferDrawer(true)
+    } else {
+      dispatch(pressSend())
+    }
+  }, [mobile, dispatch, openTransferDrawer])
+  const [, setOpen] = useModalState('MobileConnectWalletsDrawer')
+
+  const onClickConnectWallets = useCallback(() => {
+    if (mobile) {
+      setOpen(true)
+    } else {
+      dispatch(pressConnectWallets())
+    }
+  }, [mobile, setOpen, dispatch])
+
+  const onCloseConnectWalletsDrawer = useCallback(() => {
+    setOpen(false)
+  }, [setOpen])
+
+  const hasMultipleWallets = useSelector(getHasAssociatedWallets)
+  return (
+    <div className={styles.advancedSection}>
+      <div className={styles.subtitle}>{messages.advancedOptions}</div>
+      <div className={styles.advancedOptions}>
+        <Button
+          className={cn(styles.advancedButton, {
+            [styles.disabled]: !hasBalance
+          })}
+          text={messages.sendLabel}
+          isDisabled={!hasBalance}
+          includeHoverAnimations={hasBalance}
+          textClassName={styles.textClassName}
+          onClick={onClickSend}
+          leftIcon={<IconSend className={styles.iconStyle} />}
+          type={ButtonType.GLASS}
+          minWidth={200}
+        />
+        <Button
+          className={cn(styles.advancedButton, styles.receiveBtn)}
+          text={messages.receiveLabel}
+          textClassName={styles.textClassName}
+          onClick={onClickReceive}
+          leftIcon={<IconReceive className={styles.iconStyle} />}
+          type={ButtonType.GLASS}
+          minWidth={200}
+        />
+        <Button
```

Comment:
nit: This button looks a bit big on hover as well as the connect wallets button. Probably julian question. Also I realize the buttons will look different once there's also a stripe button, but the connect wallets button.

Reply:
Agreed - will want to do a follow up with design on this for sure.